sklearn/titanic/README.md

```diff
@@ -0,0 +1,138 @@
+
+##### 1. Create a Google Compute Engine Instance
+First, we should create a *GCE* instance that we can use to train our model on. In this case, we can rely on many of the default arguments:
+
+```bash
+gcloud compute instances create $MY_INSTANCE
+```
+which creates an instance of type *n1-standard-1* (1 CPU, 3.75GB of RAM), with a *Debian* disk image and no scopes. This is fine as long as we need not the instance access *GCS*. Otherwise, we will have to provide the right scope:
+
+```bash
+gcloud compute instances create $MY_INSTANCE --scopes storage-rw
+```
+
+The *scopes* argument grants the instance the right permission to access *GCS*.
+
+We can also specify a different machine type or image. For example, the following command creates an *Ubuntu* instance of type *n1-standard-2*:
+
+```bash
+gcloud compute instances create $MY_INSTANCE --machine-type=n1-standard-2 --image-family=ubuntu-1604-lts --image-project ubuntu-os-cloud --scopes storage-rw
+```
+You may want to learn more about the [machine types](https://cloud.google.com/compute/docs/machine-types) and  [images](https://cloud.google.com/compute/docs/images), or see a list of available machine types and images with the following two commands:
+
+```bash
+gcloud compute machine-types list
+gcloud compute images list
+```
+
+A more detailed instruction about *gcloud compute instance create* can be found [here](https://cloud.google.com/sdk/gcloud/reference/compute/instances/create).
+
+##### 2. Accessing and Configuring the Instance
+
+Accessing the newly created instance is quite simple:
+```bash
+gcloud compute ssh $MY_INSTANCE
+```
+This command may prompt us to create a *ssh* key, if we have not done it before on our local machine.
+
+Once we *ssh* to the instance, we will need to use *pip* to install *scikit-learn* and some other packages:
+```bash
+# Run on the new instance
+sudo apt-get install -y python-pip
+pip install numpy pandas sklearn scipy tensorflow
+```
+*Note:* If you choose a different image for the instance, you may need to do something different to install *pip* and the required python packages.
+
+Also note that we are not using *tensorflow* to train the model in this sample. We will however use *gfile*, a python class implemented in *tensorflow* which unifies how we access the local and remote files. An alternative to using *gfile*  for accessing the files stored in *GCS* is [Cloud Storage Client Libraries](https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-python).
+
+##### 3. Running the Code
+We need to make the dataset available to the training code. Here are two options for the code to access the dataset.
+
+###### Option 1: Copying it to the instance
+We can copy the dataset into the new instance and use it directly inside the instance and read it as a local file. Fortunately, *gcloud* makes this an easy step:
+```bash
+gcloud compute scp ./train.csv $MY_INSTANCE:~/
```

Comment:
There is a mention that all the commands are to be run locally, unless otherwise specified (right under ## Steps). Do you think it needs to be re-emphasized here?